lib/screenplay/screens_config/cache.ex

```diff
@@ -1,125 +1,12 @@
 defmodule Screenplay.ScreensConfig.Cache do
   @moduledoc """
-  Functions to read data from a cached copy of the screens config.
+  Cache of the Screens app config
   """
 
-  alias ScreensConfig.{Config, Devops}
+  use Nebulex.Cache,
+    otp_app: :screenplay,
+    adapter: Nebulex.Adapters.Local
 
-  use Screenplay.Cache.Client, table: :screens_config
-
-  @type table_contents :: list(table_entry)
-
-  @type table_entry ::
-          {{:screen, screen_id :: String.t()}, ScreensConfig.Screen.t()}
-          | {:devops, ScreensConfig.Devops.t()}
```

Comment:
I removed it because I can't think of a reason Screenplay would need it anyway. It was likely brought over when we changed the caching in Screenplay to match Screens and didn't think to remove it.